docs/getting-started/apostrophe-cli.md

```diff
@@ -0,0 +1,168 @@
+# The Apostrophe CLI
+
+As shown [in the previous page](creating-your-first-project.md), ApostropheCMS
+has a command line tool to help take care of potentially repetative tasks such
+as creating projects and modules. Let's take a closer look at some of the
+options. If you aren't interested in using the CLI, simply continue on to the
+next section.
+
+::: tip INFO
+As a reminder, the CLI tool is installed as a global npm module.
+
+```bash
+npm install -g apostrophe-cli
+```
+:::
+
+Let's review the commands.
+
+## Create a project
+
+To create a new project:
+```bash
+apos create-project <shortname-without-spaces>
+```
+
+This will create a local copy of our standard
+[Apostrophe Boilerplate](https://github.com/apostrophecms/apostrophe-boilerplate).
+
+`<shortname-without-spaces>` represents the "shortname" of the application. It
+will be used by default as a MongoDB database name and a basis for cookie names,
+ etc. You will find this as the `shortName` property in `app.js`.
+
+### options
+
+**`--boilerplate`**
+
+Run `create-project` with a `--boilerplate` flag to start from a Github
+repository other than the standard `apostrophe-boilerplate` repo. For example,
+to create a project using the
+[Open Museum](https://github.com/apostrophecms/apostrophe-open-museum) demo:
+
+```bash
+apos create-project my-museum-clone --boilerplate=https://github.com/apostrophecms/apostrophe-open-museum.git
+```
+
+If you run the `create-project` command with the `--setup` flag, the command
+will also `npm install` the dependencies for the project and run
+`apostrophe-users:add` to create an admin user for the CMS, resulting in a fully
+ bootstrapped project. This command will prompt you for a password for the admin
+  user being created.
+
+## Create a piece type module
+To create the necessary files and basic configuration for a new Apostrophe piece
+type, run the following command from within your Apostrophe project's root
+directory:
+
+```bash
+apos create-piece vegetable
+```
+
+You must then register `'vegetables': {}` in `apps.js` in the `modules`
```

Comment:
just saying, it's interesting here that we create 'vegetable' and then register 'vegatables'. But this is more about my pain than the docs lol.

Reply:
I mean, yes. That's why I ended up fleshing out that note about pluralization. It's something we should reconsider in the next major version. Pluralization still has a role, but there might be better ways.

Reply:
👀 for an API of all the English words that don't pluralize with "s". j/k.... maybe...